src/api/sarifViewerApi.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ */
+
+import * as vscode from 'vscode';
+
+/**
+ * Options used when the SARIF viewer is opening logs.
+ */
+export interface OpenLogArguments {
+    /**
+     * Indicates whether to prompt the user when the
+     * SARIF log schema is needs to be upgraded to the latest one schema version.
+     * The default is true.
+     */
+    readonly promptUserForUpgrade?: boolean;
+
+    /**
+     * Indicates whether to open SARIF logs inside the VSCode editor.
+     * The default is true.
+     */
+    readonly openInEditor?: boolean;
```

Comment:
After discussion with Jeff, we believe this can be dropped as we don't currently see any case where this would be passed as true.